.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: required
 dist: trusty
 git:
   depth: 9999999
+  lfs_skip_smudge: true
```

Comment:
This was the part that took me a while to figure out :) We have to disable git-lfs download for the initial git clone, since there is no way in travis to run commands before the initial clone. Then, we authenticate with github in the `before_install` phase, then run `scripts/download_lfs_files.sh` to manually download the lfs files.